packages/kit/src/core/build/index.js

```diff
@@ -179,42 +221,42 @@ async function build_client({
 
 	print_config_conflicts(conflicts, 'kit.vite.', 'build_client');
 
-	await vite.build(merged_config);
+	const output = /** @type {import('rollup').OutputChunk[]} */ (
+		/** @type {any} */ (await vite.build(merged_config)).output
```

Comment:
can this be something more specific like `RollupOutput`?

Reply:
yep, that's way nicer